apps/css-workshop/src/pages/button.astro

```diff
@@ -53,296 +54,288 @@ import Layout from './_layout.astro';
 
   <section id='demo-high-visibility'>
     <div id='demo-high-visibility-small' class='demo-container'>
-      <button class='iui-button' data-iui-variant='high-visibility' data-iui-size='small'>
+      <Button_ variant='high-visibility' size='small'>
         <span>Small</span>
-      </button>
-      <button class='iui-button' data-iui-variant='high-visibility' data-iui-size='small'>
+      </Button_>
+      <Button_ variant='high-visibility' size='small'>
         <svg-placeholder class='iui-button-icon' aria-hidden='true'></svg-placeholder>
```

Comment:
start icon and end icon need to be handled within the component. `class="iui-button-icon"` shouldn't need to be specified outside of Button, otherwise it kinda defeats the purpose of the component. This is a major oversight that I'm surprised was missed before PR merge.

This can be achieved using [named slots](https://docs.astro.build/en/basics/astro-components/#named-slots).